packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/enrollmentintegrationsteps.md

```diff
@@ -34,14 +34,14 @@ Query the `AuthenticationStatus` property of the `AuthenticationResponse` object
            … your code …
 ```
 
-If you've configured your Okta org correctly, you'll need to respond to two specific authenticator statuses to handle this scenario in addition to `Success`, `PasswordExpired` and so on:
+If you've configured your Okta org correctly, you'll need to respond to two specific authenticator statuses to handle this scenario in addition to `Success` and `PasswordExpired`:
 
 * `AwaitingAuthenticatorEnrollment` which is covered in this section
 * `AwaitingChallengeAuthenticatorSelection` which is covered in [the challenge flow section](/docs/guides/authenticators-google-authenticator/aspnet/main/#integrate-sdk-for-authenticator-challenge).
 
-The names of the available authenticators for enrollment or challenge can be found in the `AuthenticationResponse` object's `Authenticators` collection. The user should be redirected to the authenticator list page that displays the Google Authenticator factor as an authenticator to be verified.
+The names of the authenticators available for enrollment or challenge can be found in the `AuthenticationResponse` object's `Authenticators` collection. The user should be redirected to the authenticator list page that displays the Google Authenticator factor as an authenticator to be verified.
```

Comment:
@susanharper-okta But the user *is* enrolling the authenticator at this point rather than verifying their identity using the authenticator. Have reworded.